src/components/Common/ConfirmDialog.tsx

```diff
@@ -71,15 +71,18 @@ export const ConfirmDialog = ({
                 )}
               </div>
               <div className="flex items-center gap-3">
-                <button
-                  className="button button-secondary"
-                  onClick={() => {
-                    onClose(false);
-                  }}
-                  data-qa="cancel-dialog"
-                >
-                  {cancelLabel}
-                </button>
+                {cancelLabel && (
+                  <button
+                    className="button button-secondary"
+                    autoFocus
```

Comment:
I would suggest to set `autoFocus` to primary button to be able to use `Enter` to confirm and `Esc` for cancel